examples/infomax/infomax_matlab_comparison.cc

```diff
@@ -39,14 +39,14 @@ runTest(const filesystem::path &dataPath, int num)
 
     // Load matrices of weights
     const auto pref = "test"s + snum + "_"s;
-    const auto initWeights = readData<>(dataPath / (pref + "weights_init.bin"s));
-    const auto matlabOutputWeights = readData<>(dataPath / (pref + "weights_out.bin"s));
-    const auto matlabOutputWeightsMany = readData<>(dataPath / (pref + "weights_out_many.bin"));
-    const auto matlabU = readData<>(dataPath / (pref + "u.bin"s));
-    const auto matlabY = readData<>(dataPath / (pref + "y.bin"s));
+    const auto initWeights = readMatrix<double>(dataPath / (pref + "weights_init.bin"s));
```

Comment:
what do the ``s`` do?

Reply:
Makes it into a ``std::string`` rather than ``const char *``.